benchmarking/benchmark.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import pathlib
+import subprocess
+import time
+
+import openmc
+from .plot import plot
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-l', '--benchmarks', type=str,
+                        default='benchmarks/lists/pst-short',
+                        help='List of benchmarks to run.')
+    parser.add_argument('-c', '--cross-sections', type=str,
+                        default=os.getenv('OPENMC_CROSS_SECTIONS'),
+                        help='Cross sections XML file.')
+    parser.add_argument('-p', '--particles', type=int, default=10000,
+                        help='Number of source particles.')
+    parser.add_argument('-b', '--batches', type=int, default=150,
+                        help='Number of batches.')
+    parser.add_argument('-i', '--inactive', type=int, default=50,
+                        help='Number of inactive batches.')
+    parser.add_argument('-m', '--max-batches', type=int, default=10000,
+                        help='Maximum number of batches.')
+    parser.add_argument('-t', '--threshold', type=float, default=0.0001,
+                        help='Value of the standard deviation trigger on '
+                        'eigenvalue.')
+    args = parser.parse_args()
+
+    current_time = time.localtime()
+    timestamp = time.strftime("%Y-%m-%d-%H%M%S", current_time)
+
+    # Create directory and set filename for results
+    cwd = pathlib.Path(__file__).parent
```

Comment:
I would just use the current working directory the user is running from. If this Python package is installed somewhere, it will try to create directories wherever it's installed, which would be a little strange. It also seems sensible that the user would want the results in the directory they are running from.

Reply:
oh good point, that makes more sense.